server/vulnerabilities/nvd/sync/cve_syncer_test.go

```diff
@@ -95,6 +95,14 @@ func compareVulnerabilities(v1 schema.NVDCVEFeedJSON10DefCVEItem, v2 schema.NVDC
 		v.CVE.Problemtype = nil
 	}
 
+	// We now fall back to secondary CVSS score sources when primary isn't available, which legacy didn't do
```

Comment:
With this revision, match rate between old legacy and converted legacy formats remains 99.059%, so we're good here I believe. If we wanted I could revise the 2023 JSON gzip file but there are a high single digit percentage of CVEs that are affected by this change so between that and a binary diff for the gzip file I'm not sure what that buys us.